stream-chat-android-ui-common/src/main/kotlin/com/getstream/sdk/chat/viewmodel/messages/MessageListViewModel.kt

```diff
@@ -924,6 +944,30 @@ public class MessageListViewModel(
         public data class UnpinMessageError(override val chatError: ChatError) : ErrorEvent(chatError)
     }
 
+    /**
+     * Intended to be used for regulating visibility of deleted messages
+     * and filtering them out accordingly.
+     */
+    public enum class DeletedMessagesVisibility {
```

Comment:
This is potentially subject to change.

We are currently discussing naming with the iOS team in order to be synced on this matter across teams.

Reply:
Have we resolved this?

Reply:
Yes, we have synced this naming scheme with iOS.